src/main/java/ui/battle/LootView.java

```diff
@@ -93,37 +101,21 @@ public LootView(Collection<Loot> loot, Collection<EXP> exp, int playerCount) {
         Label coinLabel = new Label();
         buttonPane.setCenter(coinLabel);
 
-        Button sell = new Button("Verkaufen");
+        Button sell = new Button();
+        sell.textProperty().bind(getMessageProperty("loot.button.sell"));
         sell.setPrefWidth(100);
         buttonPane.setRight(sell);
 
         sell.setOnAction(ev -> {
-            int itemValue = 0;
-            int coinValue = 0;
-
-            for (Loot l : lootTable.getItems()) {
-                Item item = l.getItem();
-
-                if (item.getCostOfOneAsCopper() > 0) {
-                    itemValue += item.getCostOfOneAsCopper() * l.getAmount();
-                } else {
-                    switch (item.getName()) {
-                        case "Kupfer":
-                            coinValue += l.getAmount();
-                            break;
-                        case "Silber":
-                            coinValue += l.getAmount() * 100;
-                            break;
-                        case "Gold":
-                            coinValue += l.getAmount() * 10000;
-                            break;
-                    }
-                }
+            Currency sellValue = Utility.sellLoot(lootTable.getItems());
+
+            if (playerCount > 0) {
+                coinLabel.setText(sellValue.getCoinString() + "\t "+ LanguageUtility.getMessage("loot.sell.perPlayer") + ": " +
+                        sellValue.divide(playerCount).getCoinString());
+            } else {
+                coinLabel.setText(sellValue.getCoinString() );
```

Comment:
```suggestion
                coinLabel.setText(sellValue.getCoinString());
```
add a code formatter ;)

Reply:
Good idea